src/widgets/histogram/content-view.js

```diff
@@ -139,6 +139,11 @@ module.exports = cdb.core.View.extend({
       this._setupBindings();
       this.$el.toggleClass('is-collapsed', !!this.model.get('collapsed'));
       this._initViews();
+
+      if (!this.model.get('collapsed')) {
```

Comment:
Agree, and keen to end up causing infinite loops due to `change:foo` -> render -> trigger change:foo -> …